.github/workflows/ci.yml

```diff
@@ -309,10 +309,11 @@ jobs:
         set -ex
         cd src/deployment
         python -m pip install --upgrade pip
-        pip install mypy isort black types-requests
+        pip install mypy isort black types-requests flake8
         mypy .
         isort --profile black . --check
         black . --check
+        flake8 *.py
```

Comment:
Is it important that this arg is `*.py` and not `.`?

Reply:
Yes.  flake8 was picking up on the mypy cache and processing all of that content, which isn't intended.